ua-snap/data-api
# Species

This PR adds an endpoint for species data, which is accessed via shapefiles on GeoServer. Only point data queries are supported (e.g. `earthmaps.io/species/65.24/-142.22`). Documentation is not yet complete, but includes placeholder text. The description of the dataset and list of sources is in progress / pending review from project PIs. 

**TO TEST:**

- Start the `flask` application as usual.
- Navigate to the new species documentation page and review the general structure and content.
- Test some good and bad lat/lon combos, making sure you see error messages when expected.
- Test the CSV output and review the structure and content of the CSV.
